packetbeat/protos/mysql/mysql.go

```diff
@@ -18,7 +19,10 @@ import (
 
 // Packet types
 const (
-	mysqlCmdQuery = 3
+	mysqlCmdQuery       = 3
+	mysqlCmdStmtPrepare = 22
+	mysqlCmdStmtExecute = 23
+	mysqlCmdStmtClose   = 25
```

Comment:
Some more packet types to consider:

StmtBulkExecute: https://mariadb.com/kb/en/library/com_stmt_bulk_execute/
StmtFetch: https://mariadb.com/kb/en/library/com_stmt_fetch/

Reply:
I will consider these types
I did not capture any packet of these types so I just let it go：）